core/lib/refinery/plugin.rb

```diff
@@ -1,5 +1,16 @@
 module Refinery
   class Plugin
+    META = {
```

Comment:
It is a single commit, so I guess I can just point to it.  @bricesanchez I think something like this gives developers the simplicity and power to order plugins.  I've been using this for over 2 years and it has worked great for me: https://github.com/stefanspicer/refinerycms/commit/758d0ae3b0e8cc8d4bef8541d3874107a0d8f7fa